test/integration/remote-config.spec.ts

```diff
@@ -43,12 +43,12 @@ const VALID_PARAMETERS = {
 const VALID_CONDITIONS = [{
   name: 'ios',
   expression: 'device.os == \'ios\'',
-  tagColor: 'BLUE'
+  tagColor: 'INDIGO' as admin.remoteConfig.TagColor
```

Comment:
The cast shouldn't be necessary here.

Reply:
Interesting... I get a typescript compile error if I don't do the cast. Any thoughts?
```
TSError: ⨯ Unable to compile TypeScript
test/integration/remote-config.spec.ts (71,7): Type '{ name: string; expression: string; tagColor: string; }[]' is not assignable to type 'RemoteConfigCondition[]'.
  Type '{ name: string; expression: string; tagColor: string; }' is not assignable to type 'RemoteConfigCondition'.
    Types of property 'tagColor' are incompatible.
      Type 'string' is not assignable to type '"INDIGO" | "GREEN" | "BLUE" | "BROWN" | "CYAN" | "DEEP_ORANGE" | "LIME" | "ORANGE" | "PINK" | "PURPLE" | "TEAL" | undefined'. (2322)
```

Reply:
Fixed by setting the type for `VALID_CONDITIONS`!